client/my-sites/domains/domain-management/edit/gdpr-consent-status-messages.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { gdprConsentStatus } from 'lib/domains/constants';
+
+function getGdprConsentStatusMessage( domain ) {
+	let message;
```

Comment:
Also instead of having a local variable here - why not just return the translated message?

Reply:
Yep. Good point. ...more whisperings of the hobgoblin related to how `availability-messages.js` works.